presto-jmx/src/main/java/io/prestosql/plugin/jmx/JmxSplitManager.java

```diff
@@ -14,64 +14,29 @@
 import io.prestosql.spi.connector.ConnectorSplitManager;
 import io.prestosql.spi.connector.ConnectorSplitSource;
-import io.prestosql.spi.connector.ConnectorTableLayoutHandle;
+import io.prestosql.spi.connector.ConnectorTableHandle;
 import io.prestosql.spi.connector.ConnectorTransactionHandle;
 import io.prestosql.spi.connector.FixedSplitSource;
-import io.prestosql.spi.predicate.NullableValue;
-import io.prestosql.spi.predicate.TupleDomain;
-
-import javax.inject.Inject;
 
 import java.util.List;
-import java.util.Optional;
 
-import static com.google.common.base.Preconditions.checkState;
-import static io.airlift.slice.Slices.utf8Slice;
-import static io.prestosql.plugin.jmx.JmxMetadata.NODE_COLUMN_NAME;
-import static io.prestosql.spi.predicate.TupleDomain.fromFixedValues;
-import static io.prestosql.spi.type.VarcharType.createUnboundedVarcharType;
-import static java.util.Objects.requireNonNull;
 import static java.util.stream.Collectors.toList;
 
 public class JmxSplitManager
         implements ConnectorSplitManager
 {
-    private final NodeManager nodeManager;
-
-    @Inject
-    public JmxSplitManager(NodeManager nodeManager)
-    {
-        this.nodeManager = requireNonNull(nodeManager, "nodeManager is null");
-    }
-
     @Override
-    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transaction, ConnectorSession session, ConnectorTableLayoutHandle layout, SplitSchedulingStrategy splitSchedulingStrategy)
+    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transaction, ConnectorSession session, ConnectorTableHandle table, SplitSchedulingStrategy splitSchedulingStrategy)
     {
-        JmxTableLayoutHandle jmxLayout = (JmxTableLayoutHandle) layout;
-        JmxTableHandle tableHandle = jmxLayout.getTable();
-        TupleDomain<ColumnHandle> predicate = jmxLayout.getConstraint();
-
-        //TODO is there a better way to get the node column?
-        Optional<JmxColumnHandle> nodeColumnHandle = tableHandle.getColumnHandles().stream()
-                .filter(jmxColumnHandle -> jmxColumnHandle.getColumnName().equals(NODE_COLUMN_NAME))
-                .findFirst();
-        checkState(nodeColumnHandle.isPresent(), "Failed to find %s column", NODE_COLUMN_NAME);
+        JmxTableHandle tableHandle = (JmxTableHandle) table;
 
-        List<ConnectorSplit> splits = nodeManager.getAllNodes().stream()
-                .filter(node -> {
-                    NullableValue value = NullableValue.of(createUnboundedVarcharType(), utf8Slice(node.getNodeIdentifier()));
-                    return predicate.overlaps(fromFixedValues(ImmutableMap.of(nodeColumnHandle.get(), value)));
-                })
-                .map(node -> new JmxSplit(ImmutableList.of(node.getHostAndPort())))
-                .collect(toList());
+        List<ConnectorSplit> splits = tableHandle.getNodes().stream()
+                .map(node -> new JmxSplit(ImmutableList.of(node.getAddress()))).collect(toList());
```

Comment:
Move `.collect(toList());` to next line.

Reply:
> Move .collect(toList()); to next line.

Do we have a specific regulation about putting new lines? Actually, I have a similar code in `JmxMetadata` as follows. I'm wondering if I have to put new lines before `.map` and `.collect`.

```scala
private List<JmxTableHandle.Node> getAllNodes()
{
    return nodeManager.getAllNodes().stream().map(node -> new JmxTableHandle.Node(node.getNodeIdentifier(), node.getHostAndPort())).collect(toList());
}
```

Reply:
When the statement is simple, one line is fine. When you break a line after `.stream()`, subsequent each operations should be independent line as original code was. (There's no 'specific' regulation as far as I know).

The `getAllNodes` looks little long for one line and there's only one statement in the method, so putting new lines looks reasonable to me.